### DISCUSSION TITLE
挂载115出现以下提示

### DISCUSSION THREAD

**Discussion by hoylturtle** [AlistGo/alist#2204] [General]
昨天用还是好好的
今日115网盘挂载出现以下提示，更新cookies没用
![image](https://user-images.githubusercontent.com/41963499/199888991-85ae0eba-0c6a-4be6-bae3-fdf014ace6b4.png)

请问如何解决？

**Comment by jonntd** [comment]
我的没事


hoylturtle ***@***.***> 于2022年11月4日周五 12:42写道：

**Reply by dalaoJAck** [reply]
hxd，咋挂载的115啊

**Reply by jonntd** [reply]
cookie啊

**Comment by skrmc** [comment]
我也有这个问题，貌似是cookie失效以后就这样了，更新还是会报错。最近刚适配115，可能还有些小问题吧。

**Comment by SheltonZhu** [comment]
“该账号于11:57在115电脑版(windows)laptop-qtoe5pge登录。如非本人操作，请及时修改密码！”

115只能有2个设备同时在线。

新提交的代码增加了qrcode登录，修复了unicode编码问题，修复了文件list为空问题，应该也有解决这个问题，等新版本发布后在看看。

**Comment by chinh1** [comment]
![200111700-f7482d3a-74bd-4c75-9c97-b4b2c66ca65b](https://user-images.githubusercontent.com/43032941/200111764-a4764415-285f-4d16-861a-54621cb8db89.jpg)

It asks me to enter a cookie login, where do I check and add it?

No matter how much I search, I can't find the answer.

please help

**Reply by dalaoJAck** [reply]
同问，似乎没有教程

**Comment by dalaoJAck** [comment]
请问怎样挂载115啊

**Reply by y2kcan** [reply]
![QQ截图20221123135305](https://user-images.githubusercontent.com/103928359/203480556-1819b521-28e2-4027-83a8-eea6bfe12344.png)
@chinh1
@dalaoJAck

**Reply by y2kcan** [reply]
要用115浏览器1 其它浏览不行

**Comment by syboy1** [comment]
同样的问题

**Comment by SheltonZhu** [comment]
文档已经提交，等新版本吧

**Comment by iFaNGMiNGi** [comment]
cookie掉了以后 重新更新一下alist 就能解决这个问题了

**Reply by skrmc** [reply]
嗯 现在可以了
之前更换cookie还是会报异地登录

**Comment by syboy1** [comment]
前些天挂载成功，现在怎么都弄不上115，其他都没问题，而且显示还是政策的work
